ecommerce/api.py

```diff
@@ -915,3 +925,85 @@ def get_or_create_data_consents(basket):
                     ]
                 )
     return data_consents
+
+
+def create_coupons(
```

Comment:
This method has 14 parameters. Seems like a major code smell to me. I don't have any bright ideas off the top of my head for refactoring it, but worth mentioning

Reply:
I agree, I talked with @rhysyngsun about this earlier. He suggested making sure the parameters were explicit so they could be linted, but we didn't have any other ideas to improve this.